sdcm/nemesis.py

```diff
@@ -2797,7 +2797,7 @@ def modify_table_memtable_flush_period_in_ms(self):  # pylint: disable=invalid-n
             The number of milliseconds before Cassandra flushes memtables associated with this table.
             default: memtable_flush_period_in_ms = 0
         """
-        self._modify_table_property(name="memtable_flush_period_in_ms", val=random.randint(1, 5000))
+        self._modify_table_property(name="memtable_flush_period_in_ms", val=random.choice([0, random.randint(60000, 200000)]))
```

Comment:
what's the behavior in 0, it's like not setting it ? why we want to cover it ?

Reply:
0 means table with very low write rate can have memtables avoid flush for days due to low commit_log

Reply:
`N == 0` -> no periodic memtable flush
`0 < N < 6000` -> value rejected
`6000 < N` -> memtables are flushed every N milliseconds